x-pack/platform/plugins/shared/fleet/server/routes/cloud_connector/handlers.ts

```diff
@@ -76,3 +83,91 @@ export const getCloudConnectorsHandler: FleetRequestHandler<
     });
   }
 };
+
+export const getCloudConnectorHandler: FleetRequestHandler<
+  { cloudConnectorId: string },
+  undefined
+> = async (context, request, response) => {
+  const fleetContext = await context.fleet;
+  const { internalSoClient } = fleetContext;
+  const cloudConnectorId = request.params.cloudConnectorId;
+  const logger = appContextService
+    .getLogger()
+    .get('CloudConnectorService getCloudConnectorHandler');
+
+  try {
+    logger.info(`Getting cloud connector ${cloudConnectorId}`);
+    const result = await cloudConnectorService.getById(internalSoClient, cloudConnectorId);
+    logger.info(`Successfully retrieved cloud connector ${cloudConnectorId}`);
+    return response.ok({ body: result });
+  } catch (error) {
+    logger.error('Failed to get cloud connector', error.message);
+    return response.customError({
+      statusCode: 400,
+      body: {
+        message: error.message,
+      },
+    });
+  }
+};
+
+export const updateCloudConnectorHandler: FleetRequestHandler<
+  { cloudConnectorId: string },
+  undefined,
+  UpdateCloudConnectorRequest
+> = async (context, request, response) => {
+  const fleetContext = await context.fleet;
+  const { internalSoClient } = fleetContext;
+  const cloudConnectorId = request.params.cloudConnectorId;
+  const logger = appContextService
+    .getLogger()
+    .get('CloudConnectorService updateCloudConnectorHandler');
+
+  try {
+    logger.info(`Updating cloud connector ${cloudConnectorId}`);
+    const result = await cloudConnectorService.update(
+      internalSoClient,
+      cloudConnectorId,
+      request.body
+    );
+    logger.info(`Successfully updated cloud connector ${cloudConnectorId}`);
+    return response.ok({ body: result });
+  } catch (error) {
+    logger.error('Failed to update cloud connector', error.message);
```

Comment:
```suggestion
    logger.error(`Failed to update cloud connector ${cloudConnectorId}`, error.message);
```

nit: add ID to error log like the success log